api/product-sitemap.js

```
@@ -0,0 +1,66 @@
+          : "";
+        return `  <url><loc>${escapeXml(`${SITE_URL}${path}`)}</loc>${lastmod}<changefreq>weekly</changefreq><priority>0.6</priority></url>`;
+      })
+      .filter(Boolean)
+      .join("\n");
+
+    res.setHeader("Content-Type", "application/xml; charset=utf-8");
+    res.setHeader("Cache-Control", "public, s-maxage=3600, stale-while-revalidate=86400");
+    res.status(200).send(
+      `<?xml version="1.0" encoding="UTF-8"?>\n<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">\n${entries}\n</urlset>`,
+    );
+  } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(no-unused-vars)